src/kiwixapp.cpp

```diff
@@ -150,38 +150,36 @@ void KiwixApp::newTab()
 
 QString KiwixApp::findLibraryDirectory()
 {
-  // Check for library.xml in the same directory than the executable (portable kiwix-desktop)
```

Comment:
The comment is about the commit message. Please change it to "Fixed indentation and comments"